docs/diagnostic.md

```diff
@@ -0,0 +1,86 @@
+# Diagnostics
+
+One way to class AI models in Earth-2 MIP is by those that perform time-integration and those that do not.
+Diagnostic models are a class of models that are intended to transform a set of outputs from a weather/climate model into other quantities of interest.
+Functionally diagnostics are :py:class:`earth2mip.geo_operator.GeoOperator`, models can be viewed as post-processers in the inference process.
+Earth-2 MIP presently provides several built-in diagnostics.
+Those that require a model package files are hosted on [NVIDIA's model registry](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/modulus/models/modulus_diagnostics).
+
+## Motivation
+
+A natural question is: what is the point of developing a diagnostic model over a new weather/climate model that performs time integration?
+Training AI weather models is a very expensive process, requiring a large set of training data and expertise which is not always available.
```

Comment:
change to "typically requiring a large set training data, compute and expertise..."